ui/browsing/artistspage.go

```diff
@@ -111,9 +110,14 @@ func (a *ArtistsPage) Reload() {
 }
 
 func (a *ArtistsPage) load() {
-	artists, err := a.mp.GetArtists()
-	if err != nil {
-		log.Printf("error loading artists: %v", err.Error())
+	iter := a.mp.IterateArtists("")
```

Comment:
Are you planning on updating the artists page to fetch more as it's scrolled downward in a follow-up? I think GridViewPage could be used as a base for the Artists page now, but that can be a new PR.

Reply:
I plan to migrate to GridViewPage in a new PR. That will probably require more refactoring, so this should be good to go as is :)